Functions/Gherkin.ps1

```diff
@@ -397,6 +397,9 @@ function Import-GherkinFeature {
             "Scenario Outline" {
                 $Scenario = Convert-Tags -InputObject $Child -BaseTags $Feature.Tags
             }
+            "Scenario Template" {
```

Comment:
This is making me think maybe we should only look at the first word in the keyword ;-)